config/secrets.yml

```diff
@@ -1,5 +1,6 @@
 default: &default
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_token: <%= ENV["SECRET_TOKEN"] %>
```

Comment:
I need to double-check, but I think `SECRET_KEY_BASE` may have superceded `SECRET_TOKEN` in a recent-ish Rails release.